align_system/utils/outlines_prompts_utils.py

```diff
@@ -85,14 +85,20 @@ def filter_dict(d, common):
     return [filter_dict(d, common_values) for d in dicts]
 
 
-def get_relevant_structured_character_info(characters):
+def get_relevant_structured_character_info(state, actions):
```

Comment:
I don't think we can restrict the characters if an open-ended action with an unspecified character ID is available, e.g TAG_CHARACTER. Does this cover that case?

Reply:
Good point, I don't think it'd cover that case. Should I take this change out and should we just rely on TA1 to include the correct set of characters?

Reply:
Yeah, I think we should take it out, so we're not making too many assumptions